packages/palette/src/elements/Toggle/Toggle.tsx

```diff
@@ -19,6 +19,11 @@ export interface ToggleProps extends FlexProps {
   renderSecondaryAction?: (
     toggleProps: Pick<ToggleProps, "disabled" | "expanded" | "textSize">
   ) => JSX.Element
+  /**
+   * Optional onClick prop for anything that should fire in addition to toggling
+   * open/closed, e.g. tracking calls
```

Comment:
What do you mean by tracking calls here? Like tracking num of clicks?

Reply:
Oop yeah good question. I meant "tracking event calls", like the events we send to Segment. I like your version!